app/client/ui/DocTour.ts

```diff
@@ -44,7 +48,17 @@ async function makeDocTour(docData: DocData, docComm: DocComm): Promise<IOnBoard
       return String(tableData.getValue(rowId, colId) || "");
     }
     const title = getValue("Title");
-    let body: HTMLElement | string = getValue("Body");
+
+    const bodyHtmlContent = sanitizeTutorialHTML(marked.parse(getValue("Body"), {
```

Comment:
Can you use `sanitizeHTMLIntoDOM` instead? We should avoid setting `innerHTML` directly?